libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -203,8 +203,8 @@ string YulUtilFunctions::leftAlignFunction(Type const& _type)
 			break;
 		case Type::Category::Enum:
 		{
-			unsigned storageBytes = dynamic_cast<EnumType const&>(_type).storageBytes();
-			templ("body", "aligned := " + leftAlignFunction(IntegerType(8 * storageBytes)) + "(value)");
+			solAssert(dynamic_cast<EnumType const&>(_type).numberOfMembers() <= 256, "");
```

Comment:
I would find it more consistent to assert StorageBytes to be 1 here. If you check the number of members you are making assumptions about how the number of members translate to the encoding.

Reply:
Good point. Changed.